### PR TITLE
docs(readme): Fix the sample query in README

### PR DESCRIPTION
**Pull request by nikhars**
The current sample query throws a JSON error when its run on the query console. This PR fixes it.

### CODE REVIEW

**Comment by nikhars** [comment]
This actually resolves the open issue mentioned here https://github.com/getsentry/snuba/issues/1839

File: README.md [hunk 1]
```diff
@@ -108,11 +108,11 @@ An example query body might look like:
             ["tags[environment]", "=", "prod"]
         ],
         "from_date": "2011-07-01T19:54:15",
-        "to_date": "2018-07-06T19:54:15"
+        "to_date": "2018-07-06T19:54:15",
         "granularity": 3600,
         "groupby": ["group_id", "time"],
         "having": [],
-        "issues": [],
+        "issues": []
```
Comment (evanh) [inline README.md:115]: I don't think `issues` is even a valid field anymore. It would actually be nice to update this with a SnQL query example.

Comment (fpacifici) [reply README.md:115]: re: SnQL. My plan is to actually get rid of most of the README content and direct users here: https://getsentry.github.io/snuba/
once we will have a decent landing page there.
Instead use README only as an introduction like relay does.

Comment (fpacifici) [reply README.md:115]: Also, correct, please remove the issues field entirely.

Comment (nikhars) [reply README.md]: I have removed the issues field.

**Review by evanh — APPROVED** [review]

**Pull request merged**